EIPS/eip-rename_gas_to_mana.md

```diff
@@ -0,0 +1,87 @@
+---
+eip:
+title: Rename `gas` to `mana`
+description: This EIP picks up on Vitalik Buterin's (@vbuterin) EIP-102, which aimed to rename `gas` to `mana`
+author: Pascal Caversaccio (@pcaversaccio)
+discussions-to: https://ethereum-magicians.org/t/rename-gas-to-mana/13570
+status: Draft
+type: Standards Track
+category: Core
```

Comment:
This isn't a core EIP as it does not affect consensus.

Reply:
"Core: improvements requiring a consensus fork (e.g. [EIP-5](https://eips.ethereum.org/EIPS/eip-5), [EIP-101](https://eips.ethereum.org/EIPS/eip-101)), as well as changes that are not necessarily consensus critical but may be relevant to [“core dev” discussions](https://github.com/ethereum/pm) (for example, [EIP-90], and the miner/node strategy changes 2, 3, and 4 of [EIP-86](https://eips.ethereum.org/EIPS/eip-86))."

So this is nothing that is relevant to the "core dev" discussions? What is the suggested category in this case?

Reply:
Should be `interface` category. See [EIP-6](https://eips.ethereum.org/EIPS/eip-6) for a similar comparison.

Reply:
Fixed via https://github.com/ethereum/EIPs/pull/6789/commits/5ef2882980609b4edeaab70de12bdbdcebbc0ee2.